kubernetes/yamls/development/api-server/api-deployment.yml

```diff
@@ -18,13 +18,8 @@ spec:
           httpGet:
             path: /api/status.json
             port: 80
-          initialDelaySeconds: 30
-          timeoutSeconds: 3
-        readinessProbe:
-          httpGet:
-            path: /api/status.json
-            port: 80
-          initialDelaySeconds: 30
+          initialDelaySeconds: 90
+          periodSeconds: 300
           timeoutSeconds: 3
```

Comment:
Now the error I am getting is -> https://gist.github.com/vibhcool/57aaffd32707717b52745824305a37e7